### ISSUE TITLE
"django.db.utils.DatabaseError: duplicate column name: winner" when attempting to run syncdb

### ISSUE THREAD

**Issue by gunchleoc** [widelands/widelands_website#103]
After the recent updates to the website and use of newer versions of the dependencies (thanks :)), I decide to set it up from scratch again. From what I can see it works a lot better than when I wrote 737841, but there seems to be a new issue.

When running "./manage.py syncdb" I get an error message saying duplicate column name winner, which indicate some column in the database is attempted added twice. Based on the log this seemed to be related to ggz, so I commented out that module in settings.py and ran syncdb again. This time it worked without any problems, and I got the site running, so this is what is causing the problem.

**Comment by gunchleoc** [comment]
(by hjd)
Logfile from running ./manage.py syncdb complete with created tables and backtrace.

**Comment by gunchleoc** [comment]
(by sirver)
Is this from a complete new install? complete checkout, deleted the sqlite file and so on? I didn't see this error in my tests.

**Comment by gunchleoc** [comment]
(by hjd)
When setting it up again, my first step was "rm -rf ~/wlwebsite", and then follow the instructions in the readme. I have also tried removing the dev.db file and running syncdb again, and the error is reproducible.

**Comment by gunchleoc** [comment]
(by sirver)
Should be fixed in r266. Could you verify please?

**Comment by gunchleoc** [comment]
(by hjd)
The original problem looks fixed, though I get a similar error in the next table. I believe a similar fix should suffice.

I briefly tried looking at it, but I am not sure how the relations are defined between tables.

**Comment by gunchleoc** [comment]
(by sirver)
Yet another attempt is in r272.

**Comment by gunchleoc** [comment]
(by aber)
looks a little bit like the fix that I committed...

**Comment by gunchleoc** [comment]
(by sirver)
totally! The history goes like this: you commented those lines in your branch; when I merged this work I thought that was a quick fix of yours to get things working and you forgot to back it out again. So I commented them in again. Then, this bug report came up and I did a quick fix which turned out to be wrong. I then invested another hour to read up on django and came to the conclusion that your original fix was correct. "Viel Lärm um nichts".
